addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1133,7 +1133,6 @@ const ImagepickerUserValueWidget = UserValueWidget.extend({
     start: async function () {
         await this._super(...arguments);
         const allowedSelector = this.el.dataset.allowVideos;
-        this.firstFilters = (this.el.dataset.firstFilters || '').split(',').filter(s => s !== '');
```

Comment:
Technically, should be kept for custo (like your other comment says) but since we are in 13.2, let's suppose it is master: you can remove every mention of those in this branch, and hopefully no conflict in master then. Thanks !

Reply:
Doen :+1: